packages/contracts-bedrock/deploy-config/hivenet.ts

```diff
@@ -0,0 +1,6 @@
+import hivenetData from "./hivenet_data.json"
```

Comment:
You should be able to rename `hivenet_data.json` to `hivenet.json` and it should just work, see https://github.com/ethereum-optimism/optimism/blob/ac576a7e3caa552bf19968a18f30d74a485a904e/packages/hardhat-deploy-config/src/plugin.ts#L36